VideoWeb/VideoWeb/ClientApp/src/assets/i18n/en.json

```diff
@@ -395,6 +395,7 @@
     "cancel": "Cancel",
     "wingers": "Wingers",
     "panel-members": "Panel members",
+    "staff-member": "Staff Member",
```

Comment:
Do we have Welsh or is that pending?

Reply:
yeah, I will ask Lorenzo and this will be added in another ticket